examples/tutorial/03_Alignment_and_Preprocessing.ipynb

```diff
@@ -221,23 +160,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "x_center, y_center = crs.transform_point(-118.7081, 38.6942, ccrs.PlateCarree())"
+    "print((NDVI_1988.x[1] - NDVI_1988.x[0]).values)\n",
+    "print((NDVI_2017.x[1] - NDVI_2017.x[0]).values)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Then we'll select the data point nearest to this point, and use `.compute()` to actually load that data using `dask`:"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "ds.sel(x=x_center, y=y_center, method='nearest').compute()"
+    "### Select region of interest\n",
+    "\n",
+    "We'll define a central point in lat, lon convert that point to the *coordinate reference system (CRS)* of the data, and then use the area around the central point as the Region of Interest (ROI). \n",
+    "\n",
+    "The first step is getting the CRS. This information is stored in the attributed of our original landsat data. Let's take a look at it:"
```

Comment:
```suggestion
    "The first step is getting the CRS. This information is stored in the attributes of our original landsat data. Let's take a look at it:"
```

Reply:
What the heck! I guess that shows how much I rely on spellcheck.

Reply:
Unfortunately, s and d are right next to each other on the keyboard...